blocks/thought-leadership-home/thought-leadership-home.css

```diff
@@ -0,0 +1,815 @@
+/* Mobile styles and defaults */
```

Comment:
Instead of directly duplicating the styling from `blog-home.css`, let's try using the `@import` rule so that we have a more manageable stylesheet that can be used in multiple places (eg. Blog and Thought Leadership).